www/js/util.js

```diff
@@ -3120,13 +3120,67 @@ CSEmoteList.prototype.loadPage = function (page) {
             };
 
             // Add emote name
-            // TODO: editable
             var tdName = document.createElement("td");
             var nameDisplay = document.createElement("code");
             nameDisplay.textContent = emote.name;
             tdName.appendChild(nameDisplay);
             row.appendChild(tdName);
 
+            var $nameDisplay = $(nameDisplay);
+            $nameDisplay.click(function (clickEvent) {
```

Comment:
This whole thing is kind of gross and reminds me that I want to redo the JS UI rendering